vagrant-pxe-harvester/README.md

```diff
@@ -17,22 +17,29 @@ Prerequisites
 -   vagrant-libvirt plugin \>= 0.0.43.
 -   KVM (i.e. qemu-kvm), preferably the latest and greatest. This
     environment was tested with qemu-kvm 2.11.
--   Host with at least 16 CPU, 64GB RAM, and 500GB free disk space.
+-   Host with at least 16 CPU, 64GB RAM, and 500GB free disk space. (note: running this is **very** resource intensive depending on the number of nodes)
```

Comment:
:smile: I like the emphasis on: "**very**" :smile: :+1: 
As Harvester absolutely is when trying to run a full cluster virtualized, and it's great to mention that.